backend/api/utils.py

```diff
@@ -1,70 +1,45 @@
 import json
-from bson import json_util
-from typing import Any, List, Tuple, Type
+import bson
+from typing import Any
 
 import pymongo
+
+from pymongo.collection import Collection
 from rest_framework.response import Response
 from rest_framework.request import Request
 
-from .exceptions import CollectionFull
+from django.http.request import QueryDict
+
 
+DEFAULT_SORT_KEY = "id"
 
-__Collection = type(Type[pymongo.collection.Collection])
-__ApiRequest = type(Type[Request])
+DEFAULT_SORT_ORDER = pymongo.ASCENDING
 
-MIN_ID_VALUE = 100_000_000_000
-MAX_ID_VALUE = 999_999_999_999
-DEFAULT_QUERY_LIST_SORT_KEY = "id"
-DEFAULT_QUERY_LIST_SORT_ORDER = pymongo.ASCENDING
-QUERY_LIST_SORT_ORDER = {
+SORT_ORDER = {
     "ascending": pymongo.ASCENDING,
     "descending": pymongo.DESCENDING,
 }
 
 
 def _to_python_object(obj: Any) -> Any:
-    return json.loads(json_util.dumps(obj))
+    return json.loads(bson.json_util.dumps(obj))
 
 
 class JsonResponse(Response):
     def __init__(self, data: Any, *args, **kwargs):
         super().__init__(_to_python_object(data), *args, **kwargs)
 
 
-def get_sort_order_from_request(
```

Comment:
It seemed like a lot for what it was doing.

Reply:
Sometimes, I really surprise myself with my lack of efficiency, lol